state/statedb.go

```diff
@@ -209,8 +210,9 @@ func (sdb *ChainStateDB) Rollback(blockNo types.BlockNo) error {
 	sdb.Lock()
 	defer sdb.Unlock()
 
-	for sdb.latest.BlockNo > blockNo {
-		bs, err := sdb.loadBlockState(sdb.latest.BlockHash)
+	target := sdb.latest
```

Comment:
The variable name `target` looks confusing since there are two targets: the last rollback target and the final target.